server/src/controllers/eventInstanceController.ts

```diff
@@ -350,6 +350,11 @@ eventInstanceController.get(
             ticketrestrictions: {
               include: {
                 tickettype: true,
+                eventtickets: {
+                  where: {
+                    singleticket_fk: {not: null},
+                  },
+                },
```

Comment:
It looks like tickets sold of any other type also creates an `General Admission` ticket record in the `eventtickets` table which seems to throw off the count of the sold default `General Admission` tickets. What's the relation between `eventtickets` and `singletickets`? I'm not sure what the purpose of having a separate `singletickets` table is?

Reply:
Ah I see, all available tickets are added to the `eventtickets` table and a new `singleticket` is created whenever one of `eventtickets` is reserved. I'm still not sure why a `singleticket` needs to map to multiple `eventtickets`, that seems to just be throwing off the count of each tickettype sold

Reply:
That sounds good, the implementation was pretty similar to what we have now so not much needed to be changed. I made most of the changes and am just going through the backend to make sure I didn't miss anything that is effected. Then I'll do some testing and submit a PR.

Reply:
Ok, sweet, thanks @benfaustin95, should this PR go through before or after your changes?

Reply:
I think you should be able to merge your PR before the changes, the sold count for general admission will just be off temporarily.